tests/nnstreamer-edge-custom-test.c

```diff
@@ -146,37 +146,40 @@ nns_edge_custom_send_data (void *priv, nns_edge_data_h data_h)
 }
 
 static int
-nns_edge_custom_set_option (void *priv, const char *key, const char *value)
+nns_edge_custom_set_info (void *priv, const char *key, const char *value)
 {
   if (!priv || !key || !value) {
-    nns_edge_loge ("Invalid param, key(%s), value(%s)", key, value);
+    nns_edge_loge ("Invalid param, key(%p), value(%p)", key, value);
     return NNS_EDGE_ERROR_INVALID_PARAMETER;
   }
   nns_edge_custom_test_s *custom_h = (nns_edge_custom_test_s *) priv;
 
   if (strcasecmp (key, "PEER_ADDRESS") == 0) {
     SAFE_FREE (custom_h->peer_address);
     custom_h->peer_address = nns_edge_strdup (value);
-  } else {
-    nns_edge_loge ("key(%s) does not supported.", key);
-    return NNS_EDGE_ERROR_INVALID_PARAMETER;
+    return NNS_EDGE_ERROR_NONE;
   }
 
-  return NNS_EDGE_ERROR_NONE;
+  nns_edge_loge ("key(%s) does not supported.", key);
+  return NNS_EDGE_ERROR_INVALID_PARAMETER;
 }
 
-static char *
-nns_edge_custom_get_option (void *priv, const char *key)
+static int
+nns_edge_custom_get_info (void *priv, const char *key, char **value)
 {
-  if (!priv || !key) {
-    nns_edge_loge ("Invalid param, key(%s)", key);
-    return NULL;
+  if (!priv || !key || !value) {
+    nns_edge_loge ("Invalid param, key(%p), value(%p)", key, value);
```

Comment:
In the case of invalid param, it is meaningless to print the value.
```suggestion
    nns_edge_loge ("Invalid param,  custom handle, key or value shoud not be NULL.);
```